src/edu/csus/ecs/pc2/ui/AutoJudgingMonitor.java

```diff
@@ -169,7 +170,35 @@ public String getPluginTitle() {
     }
 
     
-    
+    /**
+     * Get a list of auto-judge problems the OS does not supported judging, likely due
```

Comment:
There is something wrong with the grammar in the above statement (and I can't make a suggestion on how to fix it because I'm not completely sure what it is trying to say...)

Reply:
Fixed in [this commit.](https://github.com/pc2ccs/pc2v9/pull/732/commits/a722499d915856c1364cd73c9c2eab47b4de94a6)